DB/src/test/java/io/deephaven/db/v2/select/TestFormulaColumn.java

```diff
@@ -65,6 +67,9 @@ public TestFormulaColumn(boolean useKernelFormulas) {
         availableColumns = testDataTable.getDefinition().getColumnNameMap();
     }
 
+    @Rule
+    public final DbCleanup base = new DbCleanup();
+
     @Before
     public void setUp() throws Exception {
```

Comment:
Did you mean to remove more of the setup and teardown, here?

Reply:
Yes - I didn't hit every other test class that could be slimmed down yet, trying to keep the churn down, but this one I should have removed the CompilerTools line.